debezium-testing/debezium-testing-openshift/pom.xml

```diff
@@ -0,0 +1,313 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <parent>
+    <groupId>io.debezium</groupId>
+    <artifactId>debezium-testing</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>debezium-testing-openshift</artifactId>
+  <name>Debezium OpenShift integration test-suite</name>
+
+  <properties>
+    <version.fabric8.client>4.6.4</version.fabric8.client>
+    <version.strimzi>0.16.2</version.strimzi>
+    <version.junit5>5.5.1</version.junit5>
+    <version.junit5.pioneer>0.5.1</version.junit5.pioneer>
+    <version.okhttp>4.1.0</version.okhttp>
```

Comment:
This and the next one likely already exist in the parent POM.

Reply:
True for OkHttp. AssertJ, however, isn't. Rest of the suite is using Fest. TBF I used AssertJ simply because I'm familiar with it and it's the successor to Fest.